lib/posix/sched.c

```diff
@@ -13,6 +13,16 @@
  * @brief Get minimum priority value for a given policy
  *
  * See IEEE 1003.1
+ *
+ * @param[in] policy  Scheduling policy with Zephyr thread priority ranges:
+ *	SCHED_RR:   [0, @ref CONFIG_NUM_PREEMPT_PRIORITIES -1]
+ *	SCHED_FIFO: [ @ref CONFIG_NUM_PREEMPT_PRIORITIES,
+ *		      @ref CONFIG_NUM_COOP_PRIORITIES -1]
+ *	SCHED_OTHER: [0, @ref CONFIG_NUM_PREEMPT_PRIORITIES +
+ *			 @ref CONFIG_NUM_COOP_PRIORITIES -1]
+ *
+ * @retval -1 and errno=EINVAL for a policy which is either not supported
+ *	or not allowed with Zephyr config parameters at compile time.
```

Comment:
The SCHED FIFO bit is incorrect. It's possible that the current positive-defined mapping for posix priorities is not very clear though, please see other comments.

And actually, there is no real need to doxygenate any of the internal API calls here. Their output is not processed in terms of Zephyr API pages. I think we will likely need to doxygenate headers, and IMHO, it would make more sense to do that later, all at once, using some kind of system, because there are a lot of things to doxygenate, (if that's even the route we go to generate POSIX docs).

You might as well leave this one as-is.